world_size: [5.0, 5.0]
image_size: [1000, 1000]
